dojo/user/views.py

```diff
@@ -284,7 +284,7 @@ def add_user(request):
         contact_form = UserContactInfoForm(request.POST)
         if form.is_valid() and contact_form.is_valid():
             user = form.save(commit=False)
-            user.set_unusable_password()
+            user.set_password(request.POST['password'])
```

Comment:
not sure if this is what we want / users expect. unusable password can be useful. maybe if left empty, set unusable? or have checkbox/radiobutton? or settings option?

Reply:
The goal of this PR was to get rid of the admin page. From my understanding, "set_unusable_password()" was used initially as a temporary solution when creating users.

Reply:
Also, maybe adding this to the API should be done in a different PR.

Reply:
Could you describe what problem you are trying to solve?

Reply:
The problem I am solving is getting rid of the admin page. Unusable passwords can only be reset through the admin page. With that page removed, there needs to be a way to set an initial password for a new user that can be changed through the UI, which cannot be done with unusable passwords.

Reply:
With my latest commit the Django Admin page is enabled in settings (can easily be disabled), and unusable password functionality is still available. When creating a new user, an unusable password will be set if the password field is left blank (added help text to this field on the add user page for clarification). I also tagged on a fix for a problem I discovered with the change password logic, which previously allowed the user to successfully leave both password fields blank.